swap_meet/vendor.py

```diff
@@ -1,2 +1,81 @@
 class Vendor:
-    pass
+    def __init__(self, inventory = None):
+        if inventory:
+            self.inventory = inventory
+        else:
+            self.inventory = []
```

Comment:
We can use a ternary here as well 
```suggestion
        self.inventory = inventory if inventory is not None else []
```

Reply:
Here is a good resource to learn and practice ternaries: https://www.geeksforgeeks.org/ternary-operator-in-python/